### ISSUE TITLE
ONS-International-exports-of-services-from-subnational-areas-of-the-UK failed

### ISSUE THREAD

**Issue by ajtucker** [GSS-Cogs/family-trade#180]
Jenkins job result: FAILURE
Stage: Validate CSV
[View full output](https://ci.floop.org.uk/job/GSS_data/job/Trade/job/ONS-International-exports-of-services-from-subnational-areas-of-the-UK/110/)

Blocks #24

**Comment by Shannon95** [comment]
A number of duplications and invalid_integers needs investigation.

**Comment by ajtucker** [comment]
I've created a separate issue for this in the GSS-Cogs/family-schema repository and added that this issues is blocked by it. I've moved the estimate to the other issue, as I reckon we'll have a few trade pipelines blocked by the same issue.
